tools/release/deno_workspace.ts

```diff
@@ -35,7 +35,7 @@ export class DenoWorkspace {
   getCliDependencyCrates() {
     return this.getCliCrate()
       .descendantDependenciesInRepo()
-      .filter((c) => c.name !== "test_util");
+      .filter((c) => c.name !== "tests/util/server");
```

Comment:
This will probably need to be `test_server`

Reply:
I changed to `test_server`, but wonder whether this should instead be completely reverted.

Reply:
I think you are correct -- if we are leaving the crate name this needs to be `test_util`